src/main/java/org/fcrepo/importexport/importer/Importer.java

```diff
@@ -190,20 +236,17 @@ private InputStream modelToStream(final Model model) {
         return new ByteArrayInputStream(buf.toByteArray());
     }
 
-    private URI uriForFile(final File f, final File baseDir) throws URISyntaxException {
+    private String uriPathForFile(final File f, final File baseDir) throws URISyntaxException {
         String relative = baseDir.toPath().relativize(f.toPath()).toString();
-        if (relative.startsWith("rest") && config.getResource().toString().endsWith("/rest")) {
```

Comment:
Yeah, there's a test that prevents us from importing resources that have the rdf:type fedora:RepositoryRoot that I think takes care of this... that said... I had a lot of trouble understanding what this did.